From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Wed, 30 Nov 2016 09:20:15 +0100
Subject: parse_hwdb: fix to work with pyparsing 2.1.10

pyparsing 2.1.10 fixed the handling of LineStart to really just apply to line
starts and not ignore whitespace and comments any more. Adjust EMPTYLINE to
this.

Many thanks to Paul McGuire for pointing this out!
---
 hwdb/parse_hwdb.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb/parse_hwdb.py b/hwdb/parse_hwdb.py
index 5d4c5ea..5856d12 100755
--- a/hwdb/parse_hwdb.py
+++ b/hwdb/parse_hwdb.py
@@ -56,7 +56,7 @@ except ImportError:
     lru_cache = lambda: (lambda f: f)
 
 EOL = LineEnd().suppress()
-EMPTYLINE = LineStart() + LineEnd()
+EMPTYLINE = LineEnd()
 COMMENTLINE = pythonStyleComment + EOL
 INTEGER = Word(nums)
 REAL = Combine((INTEGER + Optional('.' + Optional(INTEGER))) ^ ('.' + INTEGER))
